src/control/xojfile/XojExportHandler.h

```diff
@@ -32,6 +32,6 @@ class XojExportHandler: public SaveHandler {
     virtual void writeHeader();
     virtual void writeSolidBackground(XmlNode* background, PageRef p);
     virtual void writeTimestamp(AudioElement* audioElement, XmlAudioNode* xmlAudioNode);
-
+    virtual void writeBackgroundName(XmlNode* background, PageRef p);
```

Comment:
Please use the `override` keyword (at the end) and don't use the `virtual` keyword for these 5 functions.